pkgs/development/libraries/hunspell/dictionaries.nix

```diff
@@ -289,6 +289,39 @@ let
       };
     };
 
+  mkDictFromChromium = { shortName, dictFileName, shortDescription }:
+    stdenv.mkDerivation {
+      pname = "hunspell-dict-${shortName}-chromium";
+      version = "unstable-2023-03-08";
+
+      src = fetchgit {
+        url = "https://chromium.googlesource.com/chromium/deps/hunspell_dictionaries";
+        rev = "9e69c1c61ef631fa6406cf54887ff01de2dc6b49";
```

Comment:
Hm, I don't like hard coded Git revisions but Google doesn't really make it possible to avoid this... :o
My main issue with hard coded revisions is bitrot (when / in which intervals to update and how to get notified of new versions).
Could we maybe add an update script that picks a version based on the current stable version of Chromium that OfBorg could use to open PRs (not sure if that works but I guess it should)?
The easiest solution would probably be to reuse `pkgs/applications/networking/browsers/chromium/upstream-info.json` but there's already so much in there that it might get too out of hand - we should at least have automatic CI tests for this then.
Anyway, this is only a suggestion.

Reply:
I included an update script that updates to the version mentioned in the latest chromium version.

This currently doesn't consider changes to `dictFileName`